be-golang-plain/ocp.env

```diff
@@ -0,0 +1,2 @@
+MEMORY_LIMIT=64Mi
+MEMORY_REQUEST=32Mi
```

Comment:
Please add newlines.

Reply:
Also please note that it seems that each component must have at least 100Mi memory request.

Reply:
Done this.